docs/multicast-guide.md

```diff
@@ -0,0 +1,154 @@
+  - [Multicast NetworkPolicy statistics](#multicast-networkpolicy-statistics)
+- [Use case example](#use-case-example)
+- [Limitations](#limitations)
+  - [Encap mode](#encap-mode)
+  - [Traffic in local network control block](#traffic-in-local-network-control-block)
+  - [Linux kernel](#linux-kernel)
+<!-- /toc -->
+
+## Prerequisites
+
+Multicast support is introduced in v1.5.0 as an alpha feature. The feature gate
+`Multicast` must be enabled in the `antrea-controller` and `antrea-agent`
+configuration to use the feature, and two new configuration options -
+`multicastInterfaces` and `igmpQueryInterval` parameters are added for `antrea-agent`.
+
+```yaml
+  antrea-controller.conf: |
+    featureGates:
+    # Enable multicast traffic.
+      Multicast: true
+  antrea-agent.conf: |
+    # Enable multicast traffic.
+      Multicast: true
+    multicast:
+    # The names of the interfaces on Nodes that are used to forward multicast traffic.
+    # Defaults to transport interface if not set.
+      multicastInterfaces: 
+
+    # The interval at which the antrea-agent sends IGMP queries to Pods.
+    # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+      igmpQueryInterval: "125s"
+```
+
+## Multicast NetworkPolicy
+
+Antrea NetworkPolicy and Antrea ClusterNetworkPolicy are supported for the following
+types of multicast traffic:
+
+1. IGMP egress rules: applied to IGMP membership report and IGMP leave group messages.
+2. IGMP ingress rules: applied to IGMP query, which includes IGMPv1, IGMPv2, and IGMPv3.
+3. Multicast egress rules: applied to non-IGMP multicast traffic from the selected Pods to other Pods or external hosts.
+
+Note: multicast ingress rules are currently not supported.
+
+Examples: You can refer to the [ACNP for IGMP traffic](antrea-network-policy.md#acnp-for-igmp-traffic)
+and [ACNP for multicast egress traffic](antrea-network-policy.md#acnp-for-multicast-egress-traffic)
+examples in the Antrea NetworkPolicy document.
+
+## Debugging and collecting multicast statistics
+
+Antrea provides tooling to check multicast group information and multicast traffic statistics.
+
+### Pod multicast group information
+
+The `kubectl get multicastgroups` command prints multicast groups joined by Pods in the cluster.
+Example output of the command:
+
+```bash
+kubectl get multicastgroups
+GROUP       PODS
+225.1.2.3   default/mcjoin, namespace/pod
+224.5.6.4   default/mcjoin
+```
+
+### Inbound and outbound multicast traffic statistics
+
+`antctl` supports printing multicast traffic statistics of Pods. Please refer to the corresponding [antctl user guide section](antctl.md#multicast-commands).
+
+### Multicast NetworkPolicy statistics
+
+[The Antrea NetworkPolicyStats feature](feature-gates.md#networkpolicystats) also applies to multicast NetworkPolices.
+
+## Use case example
+
+[VLC](https://www.videolan.org/vlc/) is an open source multimedia player, it can be used for video streaming.
```

Comment:
I think we can remove the company information if Antrea is not officially claimed to being or used to be used by the peer?

Reply:
I would not talk about VLC in the first paragraph, which is just a minor thing as a tool we used in the example.

How about:
> This section will take multicast video streaming as an example to demonstrate how multicast works with Antrea. In this example, [VLC](https://www.videolan.org/vlc/) multimedia tools are used to generate and consume multicast video streams.